scripts/install/jellyfin.sh

```diff
@@ -108,219 +76,56 @@ CONFIG
-        <string>TheMovieDb</string>
-      </DisabledMetadataFetchers>
-      <MetadataFetcherOrder />
-      <DisabledImageFetchers>
-        <string>The Open Movie Database</string>
-        <string>TheMovieDb</string>
-      </DisabledImageFetchers>
-      <ImageFetcherOrder />
-    </MetadataOptions>
-  </MetadataOptions>
-  <EnableAutomaticRestart>true</EnableAutomaticRestart>
-  <SkipDeserializationForBasicTypes>false</SkipDeserializationForBasicTypes>
   <BaseUrl />
-  <UICulture>en-US</UICulture>
-  <SaveMetadataHidden>false</SaveMetadataHidden>
-  <ContentTypes />
-  <RemoteClientBitrateLimit>0</RemoteClientBitrateLimit>
-  <EnableFolderView>false</EnableFolderView>
-  <EnableGroupingIntoCollections>false</EnableGroupingIntoCollections>
-  <DisplaySpecialsWithinSeasons>true</DisplaySpecialsWithinSeasons>
-  <LocalNetworkSubnets />
   <LocalNetworkAddresses>
     <string>0.0.0.0</string>
   </LocalNetworkAddresses>
-  <CodecsUsed />
-  <IgnoreVirtualInterfaces>false</IgnoreVirtualInterfaces>
-  <EnableExternalContentInSuggestions>true</EnableExternalContentInSuggestions>
-  <RequireHttps>false</RequireHttps>
-  <IsBehindProxy>false</IsBehindProxy>
-  <EnableNewOmdbSupport>false</EnableNewOmdbSupport>
-  <RemoteIPFilter />
-  <IsRemoteIPFilterBlacklist>false</IsRemoteIPFilterBlacklist>
-  <ImageExtractionTimeoutMs>0</ImageExtractionTimeoutMs>
-  <PathSubstitutions />
-  <EnableSimpleArtistDetection>true</EnableSimpleArtistDetection>
-  <UninstalledPlugins />
 </ServerConfiguration>
 CONFIG
 #
-# Create the service file that will start and stop jellyfin.
-cat > "/etc/systemd/system/jellyfin.service" <<-SERVICE
-[Unit]
-Description=Jellyfin
-After=network.target
-
-[Service]
-User=${username}
-Group=${username}
-UMask=002
-
-Type=simple
-WorkingDirectory=$install_dir
-ExecStart=$install_dir/jellyfin -d /home/${username}/.config/Jellyfin
-TimeoutStopSec=20
-KillMode=process
-Restart=always
-RestartSec=2
-
-[Install]
-WantedBy=multi-user.target
-SERVICE
-#
 # Configure the nginx proxypass using positional parameters.
 if [[ -f /install/.nginx.lock ]]; then
     bash "/usr/local/bin/swizzin/nginx/jellyfin.sh" "${app_port_http}" "${app_port_https}"
     systemctl reload nginx
 fi
 #
 # Set the correct and required permissions of any directories we created or modified.
-chown "${username}.${username}" -R "$install_dir"
-chown "${username}.${username}" -R "$install_ffmpeg"
 chown "${username}.${username}" -R "/home/${username}/.config"
 chown "${username}.${username}" -R "/home/${username}/.ssl"
+chmod -R g+rw "/home/${username}/.config/Jellyfin"
+chmod -R g+r "/home/${username}/.ssl"
+#
+# COnfigure the jellyfin service.
+#
+systemctl stop "jellyfin.service" >> /dev/null 2>&1
+#
+cat > "/etc/systemd/system/jellyfin.service.d/jellyfin.service.conf" <<-CONFIG
+# Jellyfin systemd configuration options
+
+# Use this file to override the user or environment file location.
+
+[Service]
+#User = jellyfin
```

Comment:
Does it run as `jellyfin` if this is commented? won't it run as root otherwise?

Reply:
This is the default ENV after it;s copied and modified.

So assume yes as i'm not trying to override anything in the area.

```
# Jellyfin default configuration options
# This is a POSIX shell fragment

# Use this file to override the default configurations; add additional
# options with JELLYFIN_ADD_OPTS.

# Under systemd, use
#  /etc/systemd/system/jellyfin.service.d/jellyfin.service.conf
# to override the user or this config file's location.

#
# General options
#

# Program directories
JELLYFIN_DATA_DIR="/var/lib/jellyfin"
JELLYFIN_CONFIG_DIR="/home/username/.config/Jellyfin/config" < will be unique to the server
JELLYFIN_LOG_DIR="/var/log/jellyfin"
JELLYFIN_CACHE_DIR="/var/cache/jellyfin"

# web client path, installed by the jellyfin-web package
JELLYFIN_WEB_OPT="--webdir=/usr/share/jellyfin/web"

# Restart script for in-app server control
JELLYFIN_RESTART_OPT="--restartpath=/usr/lib/jellyfin/restart.sh"

# ffmpeg binary paths, overriding the system values
JELLYFIN_FFMPEG_OPT="--ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg"

# [OPTIONAL] run Jellyfin as a headless service
#JELLYFIN_SERVICE_OPT="--service"

# [OPTIONAL] run Jellyfin without the web app
#JELLYFIN_NOWEBAPP_OPT="--nowebclient"

#
# SysV init/Upstart options
#

# Application username
JELLYFIN_USER="jellyfin"
# Full application command
JELLYFIN_ARGS="$JELLYFIN_WEB_OPT $JELLYFIN_RESTART_OPT $JELLYFIN_FFMPEG_OPT $JELLYFIN_SERVICE_OPT $JELLYFIN_NOWEBAPP_OPT"
```

Specifically `JELLYFIN_USER="jellyfin"`

Reply:
I think those do not affect the `systemd` though, and I don't see the variable used in the `systemd` file either

Reply:
Here is the default service.

```
[Unit]
Description = Jellyfin Media Server
After = network.target

[Service]
Type = simple
EnvironmentFile = /etc/default/jellyfin
User = jellyfin
ExecStart = /usr/bin/jellyfin ${JELLYFIN_WEB_OPT} ${JELLYFIN_RESTART_OPT} ${JELLYFIN_FFMPEG_OPT} ${JELLYFIN_SERVICE_OPT} ${JELLYFIN_NOWEBAPP_OPT}
Restart = on-failure
TimeoutSec = 15

[Install]
WantedBy = multi-user.target
```

Reply:
Oh I just noticed this is an override, not the actual systemd file

Reply:
/resolved